File: src/main/java/net/minheur/potoflux/login/Account.java
package net.minheur.potoflux.login;

import net.minheur.potoflux.login.perms.Perms;

/**
 * Class storing an account
 */
public class Account {
    /**
     * The uuid of the account.
     */
    public String uuid;
    /**
     * The first name of the account.
     */
    public String firstName;
    /**
     * The last name of the account.
     */
    public String lastName;
    /**
     * The email of the account.
     */
    public String email;
    /**
     * The perms of the account.
     */
    public Perms[] perms;
    /**
     * The rank of the account.
     */
    public int rank;
    /**
     * The locked of the account.
     */
    public boolean locked;
}

File: src/main/kotlin/net/minheur/potoflux/login/Account.kt
package net.minheur.potoflux.login

import net.minheur.potoflux.login.perms.Perms

class Account {
    @JvmField
    var uuid: String? = null

    @JvmField
    var firstName: String? = null

    @JvmField
    var lastName: String? = null

    @JvmField
    var email: String? = null

    @JvmField
    var perms: Array<Perms>? = null

    @JvmField
    var rank: Int = 0

    @JvmField
    var locked: Boolean = false
}
